README.md

```diff
@@ -64,16 +64,16 @@ access fields otherwise available through `proc_get_status()`.
 ### Stream Properties
 
 Once a process is started, its I/O streams will be constructed as instances of
-`React\Stream\Stream`. Before `start()` is called, these properties are `null`.
+`React\Stream\ReadableResourceStream` as `React\Stream\WritableResourceStream`. Before `start()` is called, these properties are `null`.
 Once a process terminates, the streams will become closed but not unset.
 
 * `$stdin`
 * `$stdout`
 * `$stderr`
 
 Each of these implement the underlying
-[`DuplexStreamInterface`](https://github.com/reactphp/stream#duplexstreaminterface)
-and you can use any of its events and methods as usual:
+[`ReadableResourceStream`](https://github.com/reactphp/stream#readablestreaminterface) or [`WritableResourceStream`](https://github.com/reactphp/stream#writablestreaminterface)
```

Comment:
See above comment about `ReadableStreamInterface` and `WritableStreamInterface`. If we mention the concrete implementations, we should probably also link to their docs (eg. https://github.com/reactphp/stream#readableresourcestream) instead of the interface docs.